package.json

```diff
@@ -136,10 +136,11 @@
     "react-cookie": "^2.1.4",
     "react-final-form": "^6.3.0",
     "react-hot-loader": "^4.3.10",
-    "react-overlays": "^0.8.3",
+    "react-overlays": "3.2.0",
     "react-redux": "^5.0.2",
     "react-titled": "^1.0.0",
     "react-transform-catch-errors": "^1.0.2",
+    "react-transition-group": "2.9.0",
```

Comment:
I see that in stripes-components, react-transition-group is set to "react-transition-group": "^2.2.1".